src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.FunctionalTest/tests/TestUtil.ts

```diff
@@ -62,6 +71,8 @@ export async function waitForDocumentUpdate(
         }
     };
 
+    // Add a slight delay before checking for the first time.
+    await new Promise(r => setTimeout(r, 500));
```

Comment:
lol whaa

Reply:
Spoke offline. Without this this times out. Don't plan to spend time investigating the exact root cause right now. We'll do that later.